README.md

```diff
@@ -4,12 +4,22 @@ A Python package with basic utility functions for data science projects at Nesta
 
 ### Installation and requirements ⬇️
 
-The package requires Python 3.8, or higher. To install the package run:  
+The package requires Python 3.8, or higher. To install the basic package functionality run:
 `pip install git+https://github.com/nestauk/nesta_ds_utils.git `
 
+Since this package has functions useful for a wide range of use cases, you can choose which sets of requirements to install with the package. This is handy for not having to download large and potentially dependency-clashing packages unneccessarily.
+
+For example, if you only want to use the functions in the `/networks` folder you can run:
+`pip install git+https://github.com/nestauk/nesta_ds_utils.git#egg=nesta_ds_utils"[networks]" `
```

Comment:
I was a bit confused to find that this didn't work:
```
pip install git+https://github.com/nestauk/nesta_ds_utils.git#egg=nesta_ds_utils[networks]
```
given the discussion [here](https://github.com/pypa/pip/pull/11617). But I found it needed the quotation marks (and outside the square brackets too). 🤷‍♀️